style.css

```diff
@@ -123,72 +178,133 @@ hr {
+
 .grupo-imagens-projetos {
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
 }
 
-.meu-corretor {
+.propriedades-img-projetos {
   width: 8rem;
   height: 8rem;
-  padding: 0.1rem;
-  background: linear-gradient(to bottom, #ff01d538, rgba(253, 78, 241, 0.507)),
-      url("assets/meucorretor.png");
+  padding: 0.2rem;
+  font-family: 'Source Code Pro', monospace;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  color: rgb(255, 255, 255);
+  text-decoration: none;
+  text-transform: uppercase;
+  margin: 0.5rem;
+  border-radius: 5px;
+}
+
+.propriedades-img-projetos:hover {
+  transition: 0.3s;
+  filter: grayscale(70%);
+}
+
+.meu-corretor {
+  background: linear-gradient(to bottom, #5207464e, rgba(102, 9, 96, 0.601), rgba(70, 18, 66, 0.507)),
+    url("assets/meucorretor.png");
   background-size: 8rem;
-} 
+}
 
 .calculadora {
-  width: 8rem;
-  height: 8rem;
-  padding: 0.1rem;
-  background: linear-gradient(to bottom, #ff01d538, rgba(253, 78, 241, 0.507)),
-      url("assets/calculadoraimc.png");
+  background: linear-gradient(to bottom, #5207464e, rgba(102, 9, 96, 0.601), rgba(70, 18, 66, 0.507)),
+    url("assets/calculadoraimc.png");
   background-size: 8rem;
-} 
+}
 
-.estagio-globo, .estagio-buser {
-	color: #4f6068;
+.portifolio {
+  background: linear-gradient(to bottom, #5207464e, rgba(102, 9, 96, 0.601), rgba(70, 18, 66, 0.507)),
+    url("assets/portifolio.png");
+  background-size: 8rem;
+}
+
+.enzo {
+  background: linear-gradient(to bottom, #5207464e, rgba(102, 9, 96, 0.601), rgba(70, 18, 66, 0.507)),
+    url("assets/enzo.png");
+  background-size: 8rem;
+}
+
+.order {
+  background: linear-gradient(to bottom, #5207464e, rgba(102, 9, 96, 0.601), rgba(70, 18, 66, 0.507)),
+    url("assets/sound.png");
+  background-size: 8rem;
+}
+
+.roger {
+  background: linear-gradient(to bottom, #5207464e, rgba(102, 9, 96, 0.601), rgba(70, 18, 66, 0.507)),
+    url("assets/siteroger.png");
+  background-size: 8rem;
+}
+
+.site-buser {
+  background: linear-gradient(to bottom, #5207464e, rgba(102, 9, 96, 0.601), rgba(70, 18, 66, 0.507)),
+    url("assets/pagbuser.png");
+  background-size: 8rem;
+}
+
+.pag-qr-code {
+  background: linear-gradient(to bottom, #5207464e, rgba(102, 9, 96, 0.601), rgba(70, 18, 66, 0.507)),
+    url("assets/qr.png");
+  background-size: 8rem;
+}
+
+
+
+.estagio-globo,
+.estagio-buser {
+  color: #4f6068;
 }
 
 .estagio-globo {
-  margin-top:  2rem;
+  margin-top: 2rem;
 }
 
 .estagio-buser {
-	margin-bottom: 2rem;
+  margin-bottom: 2rem;
 }
 
-.imagem-experiencia{
-	width: 20px;
-	height: 20px;
+.imagem-experiencia {
+  width: 20px;
+  height: 20px;
 }
 
-.titulo-buser, .titulo-globo, .titulo-etec, .titulo-unesp, .titulo-boticario, .titulo-unifesp, .titulo-busertech {
-	display: flex;
-	flex-direction: row;
-	align-items: center;
+.titulo-buser,
+.titulo-globo,
+.titulo-etec,
+.titulo-unesp,
+.titulo-boticario,
+.titulo-unifesp,
+.titulo-busertech {
```

Comment:
que tal unificar em `titulo-formacao`? Imagina que a pessoa que for dar manutenção, toda vez que ele ver um título ela vai consultar o que é. <br>
Em vez dela fazer isso 6 vezes, ela só vai ver uma vez.

Reply:
achei top